### DISCUSSION TITLE
Best way to add a new partial/section on a blog post

### DISCUSSION THREAD

**Discussion by mirontoli** [gohugo-ananke/ananke#521] [2 - Q&A / Support] [answered]
Hi!

What is the best way to add a section to a single.html?

![image](https://user-images.githubusercontent.com/470105/155033702-f78c1759-391c-41c1-bb5f-bfd14bd7ce91.png)

A wordier version of my question: I have moved my blog from wordpress.com to a static site. Something that I am missing a bit is comments. I have tried disqus, but it was too much ads and too much traffic and network payload. So I ended up with an idea of skipping comments alltogether. 

What I would like to to is to add a section on every blog post just where disqus or commentto would be - a message that says something like this: "*Want to comment or discuss? Share it on LinkedIn or Twitter and come into discussion*". 

My question is about guidance. To add a new a section, should I update the Ananke page (themes/ananke/layouts/_default/single.html)? Or is there a way to add it without touching the Ananke code, so that I don't get conflicts when I update it? Or are there more alternatives that I don't see yet? 

Thank you very much in advance.

**Comment by RoninTech** [comment, marked as answer]
If you want to modify Ananke, the best way is to copy the file from the theme folder to the root layouts folder.  So if you want to modify themes/ananke/layouts/_default/single.html copy it to layouts/_default/single.html and make your edits.  If a file exists in the base layouts folder it will override the same file in your theme folder.
